src/proxy/query.go

```diff
@@ -342,9 +345,16 @@ func queryStat(node sqlparser.Statement, err error) {
 	default:
 		command = "Unsupport"
 	}
+	queryTime := time.Since(timeStart)
 	if err != nil {
+		if queryTime > slowQueryTime {
+			monitor.SlowQueryTotalCounterInc(command, "Error")
+		}
 		monitor.QueryTotalCounterInc(command, "Error")
 	} else {
+		if queryTime > slowQueryTime {
+			monitor.SlowQueryTotalCounterInc(command, "OK")
```

Comment:
the same as upper

Reply:
The same as upper, but I`ll try to add more test to cover thin problem.